packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -26,7 +26,7 @@ export const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
 
 export const ERC20TokenAddressL1 = '0x326C977E6efc84E512bB9C30f76E30c160eD06FB'
 export const ERC20TokenAddressL2 = '0xd14838A68E8AFBAdE5efb411d5871ea0011AFd28'
-
+export const unbridgedERC20Token = '0xcfDE5cb8bF1071584d346220317c4d94c2Bc09cD'
```

Comment:
NIT : `unbridgedERC20Token` -> `UnbridgedERC20Token`. Consistent with the 2 constants above.